libs/shared/src/consts/handlebar-helpers/getTemplateVariables.ts

```diff
@@ -10,27 +10,48 @@ export interface IMustacheVariable {
 }
 
 export function getTemplateVariables(bod: any[]): IMustacheVariable[] {
+  const pairVariables = bod
+    .filter((body) => body.type === 'HashPair')
+    .flatMap((body) => {
+      const varName = body.value?.original as string;
+
+      if (!shouldAddVariable(varName)) {
+        return [];
+      }
+
+      return {
+        type: TemplateVariableTypeEnum.STRING,
+        name: body.value?.original as string,
+        defaultValue: '',
+        required: false,
+      };
+    });
+
   const stringVariables: IMustacheVariable[] = bod
     .filter((body) => body.type === 'MustacheStatement')
     .flatMap((body) => {
       const varName = body.params[0]?.original || (body.path.original as string);
 
-      if (body.path.original === HandlebarHelpersEnum.I18N) {
+      if (body.path?.original === HandlebarHelpersEnum.I18N) {
+        if (body.hash?.pairs) {
+          return getTemplateVariables(body.hash.pairs);
```

Comment:
I'm not sure if we need a recurrent call here? 
Can't we just use the extracted above function (lines 13-29) to get the variables?

Reply:
Currently, yes, you are right. But when we have time, in the cooldown maybe I want to go back to it and add support for sub expressions. In i18n, but also for general use. And that will need the recurrent calls.
For example:
`{{titlecase (uppercase msg) }}` (not a useful example, but just to show my point)
or for i18n 
`{{i18n "group.key" var=(titlecase msg) }}`

`msg` in both examples would not be recognized as a variable, and in the first example it would set `uppercase` as a variable instead. 

I do have something working for those states, but it requires some more work. I didn't want to over do it with the time as I feel its not the biggest priority

Reply:
ok, makes sense now :)